### ISSUE TITLE
Fix quintic walk build

### ISSUE THREAD

**Issue by texhnolyze** [bit-bots/bitbots_meta#137]
`bitbots_quintic_walk` build issues:
```
--- stderr: bitbots_moveit_bindings
CMake Error at CMakeLists.txt:15 (find_package):
  By not providing "Findbio_ik_msgs.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "bio_ik_msgs", but CMake did not find one.
```
Due to f363f15 in bitbots_meta, where `bio_ik_service` was removed, which contained the `bio_ik_msgs` module.

**Comment by SammyRamone** [comment]
True we do need the bio_ik_msgs package. So it should go back into meta. But we should also remove the other packages in this git since they are not building https://github.com/bit-bots/bio_ik_service/pull/2

**Comment by jaagut** [comment]
Resolved by #139
